test/lib/ufe/testComboCmd.py

```diff
@@ -613,7 +613,7 @@ def testFallbackCases(self):
                 "xformOp:rotateZ", "!invert!xformOp:translate:pivot",
                 "xformOp:rotateXYZ:maya_fallback")))
 
-    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 121 or platform.system() == 'Linux', 'Fallback transform op handling only available in Maya Preview Release 121 or later.')
+    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 122 , 'Fallback transform op handling only available in Maya Preview Release 121 or later.')
```

Comment:
You'll need to adjust the message...  Still says 121.  Maybe something like:
"This test requires Maya functionality only available in Maya Preview Release 122 or later."

Reply:
@ppt-adsk Ah Good point... I finally got the message correct. lol

See eb064a5b1dfcccfaa8f232bb081e131df073593b